packages/office-ui-fabric-react/src/components/__snapshots__/Tooltip.Overflow.Example.tsx.shot

```diff
@@ -115,7 +115,7 @@ exports[`Component Examples renders Tooltip.Overflow.Example.tsx correctly 1`] =
         role="switch"
         type="button"
       >
-        <div
+        <span
```

Comment:
no clue

Reply:
Oh that's my PR #10901 conflicting with yours #11445...I added a toggle to an example and you changed the toggle styles. I fixed it in #11452.

Reply:
I mean I opened the fix...please feel free to approve and merge. :)